scripts/release/update-sdk-ts-defs.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @fileoverview
+ * Updates the SDK Typescript definitions located in the file sdk.d.ts
+ * to match the current definitions of functions of sdk.mjs.
+ */
+
+import fs from 'node:fs';
```

Comment:
```suggestion
import fs from 'node:fs/promises';
```

We can delightfully use [File System Promises API](https://nodejs.org/api/fs.html#promises-api) and top-level-await.